run.py

```diff
@@ -114,11 +109,12 @@ async def getBriefing():
             for u in url_list:
                 url += u.group(1)
             url = url.replace('http', 'https')
-            # print(url)
+            print(url)
         async with session.get(url=url, params=params, ssl=False, cookies=cookies, headers=headers) as resp:
             text = await resp.text()
-            bs = BS(text, "html.parser")
```

Comment:
Thanks for your contribution! I really appreciate your help to fix this code. And it seems here is the key to this problem. The webpage of wechat article has changed the way to add news image: it now loads it using js below:

```html
<div class="share_media" id="img_list">
<script nonce="1540600482" reportloaderror="">
            if (!window.__second_open__) {
              var l = '', w, h, r;
              var c = document.getElementById('img_list');
              var cw = c.offsetWidth;
              var ls = 'data:image/gif;base64,iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAYAAAAfFcSJAAAADUlEQVQImWNgYGBgAAAABQABh6FO1AAAAABJRU5ErkJggg==';
                            w = "1000" * 1 || 0;
              h = "6140" * 1 || 0;
              r = (w && h / w || 0) * cw + 'px' || 'auto';
              l += '<img data-src="https://mmbiz.qpic.cn/mmbiz_jpg/F7bGKvqEOzD6ZHgndmyXooKFibRR2JtN7koMKeNCDdxZAVXONFzddG9SCMkiaUYJ5JnqL7c3EP1f9b2kULXmeG6w/0?wx_fmt=jpeg" src="' + ls + '" style="height:' + r + '">';
                            c.innerHTML = l;
            }
            </script>
</div>
```

I would keep this change. Additionally, I notice that you updated the scheduled task time to the exact time when the article is published from wechat, i.e., 8am UTC+8. Since the code is retrieving content form sougou instead of wechat. Sometimes it may cause failure as there is a delay between two platforms. As a result, I prefer the original time which is 10:30am UTC+8.

And regards with other changes, I am afraid I cannot accept them as they are not closely related to the issue of action failure. 

Once again, thanks for your fix!

Reply:
By the way, if you want your code to be merged, could you please clean all the other changes temporarily except the key change? so I can merge it easily. And later you can apply whatever changes you want in your own repository.

If you would not like your code to be merged, you are free to to close this pull request and I am going to copy your fix to my repository and fix the problem.

Reply:
I've clean all the other changes except the key one. Besides, I noticed that the way of how webpage article in wechat render pics changes again,now it looks like this:
```
<script type="text/javascript" nonce="1482281912" reportloaderror>
(function() {
  if (!!window.__initCatch) {
    window.__initCatch({
      idkey: 66881,
      startKey: 30,
      limit: 10,
      reportOpt: {
        uin: "",
        biz: "",
        mid: "",
        idx: ""
      }
    });
  }
  // 评论数据上报
  window.pageCommentReportData = {
    scene : 3, // 图片分享页
    idkey : 66881,
    moreList : 31, // 加载第二页或以上的留言列表次数
    errList : 32, // 加载评论列表失败
    addCommentErr : 33, // 发表评论失败
    // 加载我的评论失败
    errComment : 34
  };
  window.cgiData = {
    nick_name: "易即今日",
    round_head_img : "http://mmbiz.qpic.cn/mmbiz_png/F7bGKvqEOzC8jmuZz6iaytmXPibVUzpjAbV3nCBTRuU04SHicMYGlrJxrlzJaJaHicww7CHY74fpXf8HO0fw1qB2WQ/0?wx_fmt=png",
    user_name: "gh_f3e379080862",
  };
  window.picture_page_info_list = [
          {
        width: '1000' * 1,
        height: '6140' * 1,
        cdn_url: 'https://mmbiz.qpic.cn/mmbiz_jpg/F7bGKvqEOzD0ziafkz5Yvhfmnds4uy7fmm1vJ7mhr5srG0GekKwwWXKFURrOEicNiaqESXickRFQJ53qImK5nCp6iaQ/0?wx_fmt=jpeg',
      },
      ];
  window.from_masonry = "" * 1;
  window.new_appmsg = 1;
  window.name = "易即今日";
  window.desc = "今日简报（8月11日）".replace(/\r/g, '').replace(/\n/g, '<br>').replace(/\s/g, '&nbsp;');
})();
</script>
```
I changed the code accordingly,and hopefully you'd accept it☺️